### PR TITLE
NEW CFD-460 : Comments are not truncated on the homepage activity stream

### PR DESCRIPTION
**Pull request by solker**
Now comments truncated for next types:
* discussion
* event
* photo
* photoalbum
* document
* wiki_page
* task
* tasklist
* article

selection_012
selection_013

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@solker Looks good. Assign to Marteen as soon as test are green.

**Comment by HelenaEksler** [comment]
@MPParsley @zero2one PR is ready for review.

File: capacity4more/modules/c4m/content/c4m_content_tasklist/c4m_content_tasklist.info [hunk 1]
```diff
@@ -18,13 +19,19 @@ dependencies[] = og
...
 dependencies[] = taxonomy
 dependencies[] = text
+features[ctools][] = ds:ds:1
 features[ctools][] = strongarm:strongarm:1
+features[ds_field_settings][] = comment|comment_node_tasklist|activity_stream
+features[ds_layout_settings][] = comment|comment_node_tasklist|activity_stream
 features[features_api][] = api:2
 features[field_base][] = c4m_related_tasklist
 features[field_base][] = c4m_tasklist_status
+features[field_base][] = comment_body
```
Comment (MPParsley) [inline capacity4more/modules/c4m/content/c4m_content_tasklist/c4m_content_tasklist.info]: This should be featured in c4m_comment

File: capacity4more/modules/c4m/content/c4m_content_tasklist/c4m_content_tasklist.features.field_base.inc [hunk 2]
```diff
@@ -72,5 +72,34 @@ function c4m_content_tasklist_field_default_field_bases() {
...
+        0 => 'format',
+      ),
+    ),
+    'locked' => 0,
+    'module' => 'text',
+    'settings' => array(),
+    'translatable' => 0,
+    'type' => 'text_long',
+  );
+
```
Comment (MPParsley) [inline capacity4more/modules/c4m/content/c4m_content_tasklist/c4m_content_tasklist.features.field_base.inc]: Please move this to c4m_comment.

**Comment by solker** [comment]
@MPParsley fixed.

**Pull request merged**